templates/CRM/common/TabSelected.tpl

```diff
@@ -0,0 +1,12 @@
+<script type='text/javascript'>
+  var selectedTab = '{$defaultTab}';
+  var tabContainer = '#mainTabContainer';
+  {if $tabContainer}tabContainer = '{$tabContainer}';{/if}
+  {if $selectedChild}selectedTab = '{$selectedChild}';{/if}
+  {literal}
+  CRM.$(function($) {
+    var tabIndex = $('#tab_' + selectedTab).prevAll().length
```

Comment:
I think there should be a semicolon at the end of this line -for style even if it works without